src/ert/_c_wrappers/config/rangestring.py

```diff
@@ -78,7 +89,16 @@ def rangestring_to_mask(rangestring: str, length: int) -> List[bool]:
 
 def rangestring_to_list(rangestring: str) -> List[int]:
     """Convert a string specifying ranges of elements, and the number of elements,
-    into a list of ints. The ranges are end-inclusive."""
+    into a list of ints. The ranges are end-inclusive.
+
+    >>> rangestring_to_list("0-1")
+    [0, 1]
```

Comment:
Would it be a good idea to replace 
```
>>> rangestring_to_list("0-1")
    [0, 1]
```
with 
```
>>> rangestring_to_list("0-1, 4-6, 8")
    [0, 1, 4, 5, 6, 8]
```
just to capture that multiple ranges can be in the input string

Reply:
Done :)